crit-go/cmd/root.go

```diff
@@ -0,0 +1,38 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+	"os"
+)
+
+var inloc, outloc, dir, what, cfgFile string
+var pretty, nopl bool
+
+// rootCmd represents the base command when called without any subcommands
+
+var rootCmd = &cobra.Command{
+	Use:   "crit-go",
+	Short: "CRIT is a feature-rich replacement for existing criu show",
+	Long: `CRIT is a feature-rich replacement for existing "criu show". 
+	This version is written in Go for usage with Go codebase, crit is also available in python
+	for more information visit https://criu.org/CRIT `,
+	Run: func(cmd *cobra.Command, args []string) {
+		fmt.Println("crit executed")
```

Comment:
all errors and debug messages have to be printed on stderr. I would recomend to use one of logging libraries instead of using fmt.Println.

Reply:
> > Sure, what do you mean by update them though?
> 
> I mean we can add a new image in CRIU or we can add a new field in an existing image and then we will need to update pb.go files in this project and we need an instruction how to do this. In ideal case, we need a script while will do this.

there is already a pr to update the images,which uses the go-out flag so that we can generate these using make on the system itself.

Reply:
is there a recommended logging library for that criu uses or can i use any logging lib ?

Reply:
refactored to use logging